libbzip2-rs-sys/src/bzlib.rs

```diff
@@ -162,6 +162,22 @@ impl bz_stream {
         }
         Some(b)
     }
+
+    fn write_byte(&mut self, byte: u8) -> bool {
```

Comment:
mark this (and the `read_byte` from earlier) as `#[must_use]`? not using this value is almost certainly a bug.